include/cantera/clib/ctfunc.h

```diff
@@ -15,7 +15,13 @@ extern "C" {
 #endif
 
     CANTERA_CAPI int func_new(int type, size_t n, size_t lenp, const double* p);
+    CANTERA_CAPI int func_new_essential(const char* type, double c);
+    CANTERA_CAPI int func_new_advanced(
+        const char* type, size_t lenp, const double* p, size_t n);
+    CANTERA_CAPI int func_new_compound(const char* type, size_t a, size_t b);
+    CANTERA_CAPI int func_new_modified(const char* type, size_t a, double c);
```

Comment:
Sorry for not catching this before, but since the handles returned by clib are `int`s, then the handle arguments here should be `int`s as well.